src/npctalk_funcs.cpp

```diff
@@ -741,6 +742,30 @@ void talk_function::buy_100_logs( npc &p )
     add_msg( m_good, _( "%s drops the logs off in the garage…" ), p.get_name() );
 }
 
+/*
+ * Function to make the npc drop non favorite, worn or wielded items at their current position.
+ */
+void talk_function::drop_items_in_place( npc &p )
+{
+    std::vector<drop_or_stash_item_info> to_drop;
+
+    // add all non favorite carried items to the drop off list
+    for( const item_location &npcs_item : p.all_items_loc() ) {
+        if( npcs_item.where() != item_location::type::character && !npcs_item->is_favorite ) {
+            to_drop.emplace_back( npcs_item, npcs_item->count() );
+        }
+    }
+    if( !to_drop.empty() ) {
+        // spawn a activity for the npc to drop the specified items
+        p.assign_activity( player_activity( drop_activity_actor(
+                                                to_drop, tripoint_zero, false
+                                            ) ) );
+        add_msg( m_good, _( "%s will drop carried items." ), p.get_name() );
+    } else {
+        add_msg( m_info, _( "%s has nothing to drop." ), p.get_name() );
```

Comment:
```suggestion
        p.say( "<acknowledged>" );
    } else {
        p.say( _("I don't have anything to drop off.") );
```

What you wrote is functional, but you can make it more immersive by having the character talk back instead of silently following orders.

Reply:
yeah i guess i need to learn and think more about that this is a game you play :D not a software you operate ^^
also i guess those <x> stand for placeholders? I saw it also for swearing and such?